src/app.js

```diff
@@ -1,21 +1,130 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const fs = require('fs');
+const path = require('path');
+
+const terminal = process.argv.slice(2);
+
+function renameFile() {
```

Comment:
Function renameFile should have parameters instead of using variables from outer scope. It's better to pass terminal array as an argument.